osu.Game/Screens/OnlinePlay/Match/RoomSubScreen.cs

```diff
@@ -86,7 +86,7 @@ protected override BackgroundScreen CreateBackground() => new RoomBackgroundScre
         public readonly Room Room;
         private readonly bool allowEdit;
 
-        private ModSelectOverlay userModsSelectOverlay;
+        public ModSelectOverlay UserModsSelectOverlay { get; private set; }
```

Comment:
Could probably mark this as `internal` (I think we've done that elsewhere to mark that it's only "public" for testing).